fuel-core/src/executor.rs

```diff
@@ -40,10 +46,21 @@ impl Executor {
             self.persist_owners_index(block.fuel_height, &tx, tx_id, idx, block_tx.deref_mut())?;
 
             // execute vm
-            let mut vm = Interpreter::with_storage(tx_db.clone());
-            let execution_result = vm.transact(tx);
-
-            match execution_result {
+            let mut vm = Transactor::new(tx_db.clone());
+            vm.transact(tx);
+            if config.backtrace {
```

Comment:
Maybe move this to here so we only log backtraces on an error: https://github.com/FuelLabs/fuel-core/pull/79/files#diff-e7bf9ceaafdbbc3186af806322dfb8656d1dccae8291627e8e470fbbb15781dcL56

Reply:
Makes sense, I think this is even behaviour that is expected for backtrace.
Pushed commit that moves it inside revert.